code/modules/client/verbs/autobunker.dm

```diff
@@ -0,0 +1,40 @@
+/client/verb/bunker_auto_authorize()
+	set name = "Auto Authorize Panic Bunker"
+	set desc = "Authorizes your account in the panic bunker of any servers connected to this function."
+	set category = "OOC"
+	
+	var/static/lastuse = 0
+	if(lastuse + 5 SECONDS > world.time)
+		to_chat(src, "<span class='danger'>Function on cooldown, try again in 5 seconds.</span>")
+		return
+	lastuse = world.time
+
+	world.send_cross_server_bunker_overrides(key, src)
+
+/world/proc/send_cross_server_bunker_overrides(key, client/C)
+	var/comms_key = CONFIG_GET(string/comms_key)
+	if(!comms_key)
+		return
+	var/list/message = list()
+	message["ckey"] = key
+	message["source"] = "[CONFIG_GET(string/cross_comms_name)]"
+	message["key"] = comms_key
+	message["auto_bunker_override"] = TRUE
+	var/list/servers = CONFIG_GET(keyed_list/cross_server_bunker_override)
+	if(!length(servers))
+		to_chat(C, "<span class='boldwarning'>AUTOBUNKER: No servers are configured to receive from this one.</span>")
+		return
+	var/logtext = "[key] ([key_name(C)]) has initiated an autobunker authentication with linked servers."
+	message_admins(logtext)
```

Comment:
...And harrass admins about it every 5 seconds too. Perhaps only warn them on success.,,

Reply:
it's logged + message so admins know when someone is doing it 

admins can tell them to stop spamming 

hm

Reply:
I'm unwilling to push the need of constant server and game flow monitoring from admins furthermore.

Reply:
yeah sure